src/MCPClient/lib/clientScripts/archivematicaCreateMETS2.py

```diff
@@ -827,7 +829,59 @@ def createFileSec(directoryPath, parentDiv, baseDirectoryPath, baseDirectoryName
             file_elem.set("DMDID", dspaceMetsDMDID)
     
     return structMapDiv
-        
+
+def build_arranged_structmap(original_structmap, fileGroupIdentifier):
```

Comment:
Docstring?  Especially coming to this after a year I'm not sure what this function is supposed to do.